Formula/totp-cli.rb

```diff
@@ -0,0 +1,20 @@
+class TotpCli < Formula
+  desc "Authy/Google Authenticator like TOTP CLI tool written in Go"
+  homepage "https://yitsushi.github.io/totp-cli/"
+  url "https://github.com/yitsushi/totp-cli/archive/v1.2.4.tar.gz"
+  sha256 "0451172dbd85ecddbe5faf14fecd7cefab3e8678ff6b8d6f5cb6cfa8d7defa3b"
+  license "MIT"
+  head "https://github.com/yitsushi/totp-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match "totp-cli v", shell_output("#{bin}/totp-cli version")
```

Comment:
Let's check this instead, go builds often get the version information mixed up
```suggestion
    assert_match "totp-cli v#{version}", shell_output("#{bin}/totp-cli version")
```

Reply:
Good catch, I actually had tried that, however there's a bugfix which corrected the version after totp-cli 1.2.4 was released :-) See https://github.com/yitsushi/totp-cli/commit/f8a7bceff146d55d4f21fb7a392051cda3d6ae1f, due to which we can't verify the exact version, at least for now.